src/components/Icon/Email.tsx

```diff
@@ -0,0 +1,12 @@
+import  React from 'react'
+
+export default React.memo(function Email(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
We don't need to memo the icons. I'd also avoid to pass all props and instead limit what props we need for icons.